tests/TestCase/Console/HelpFormatterTest.php

```diff
@@ -442,28 +445,28 @@ public function testXmlHelpWithOptions()
         $expected = <<<xml
 <?xml version="1.0"?>
 <shell>
-<name>mycommand</name>
+<command>mycommand</command>
 <description/>
 <subcommands/>
 <options>
-	<option name="--help" short="-h" help="Display this help." boolean="1">
-		<default></default>
+	<option name="--connection" short="-c" help="The connection to use." boolean="">
```

Comment:
Is there a way we could retain the `0`? Booleans in XML are awkward, but ''/0 is more difficult than 1/0